src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/PeopleInControl/Orchestrator/PeopleInControlTests.cs

```diff
@@ -0,0 +1,175 @@
+                    Name =personInControlName + directorsPostfix + sourceSubmitted,
+                    DateOfBirth = personInControlDob + directorsPostfix + sourceSubmitted
+                } };
+            _organisationSummaryApiClient.Setup(x => x.GetDirectorsFromSubmitted(applicationId)).ReturnsAsync(directorsFromSubmitted);
+
+            var pscsFromCompaniesHouse = new List<PersonInControl>
+            {
+                new PersonInControl
+                {
+                    Name = personInControlName+pscPostfix + sourceExternal,
+                    DateOfBirth = personInControlDob + pscPostfix + sourceExternal
+                }
+            };
+            _organisationSummaryApiClient.Setup(x => x.GetPscsFromCompaniesHouse(applicationId)).ReturnsAsync(pscsFromCompaniesHouse);
+
+            var pcsFromSubmitted = new List<PersonInControl>
+            {
+                new PersonInControl
+                {
+                    Name = personInControlName+pscPostfix + sourceSubmitted,
+                    DateOfBirth = personInControlDob + pscPostfix + sourceSubmitted
+                }
+            };
+            _organisationSummaryApiClient.Setup(x => x.GetPscsFromSubmitted(applicationId)).ReturnsAsync(pcsFromSubmitted);
+
+
+            var trusteesFromCharityCommission = new List<PersonInControl>
+            {
+                new PersonInControl
+                {
+                    Name = personInControlName+ trusteesPostfix + sourceExternal
+                }
+            };
+            _organisationSummaryApiClient.Setup(x => x.GetTrusteesFromCharityCommission(applicationId)).ReturnsAsync(trusteesFromCharityCommission);
+
+            var trusteesFromSubmitted = new List<PersonInControl>
+            {
+                new PersonInControl
+                {
+                    Name = personInControlName+ trusteesPostfix + sourceExternal,
+                    DateOfBirth = personInControlDob + trusteesPostfix + sourceExternal
+                }
+            };
+            _organisationSummaryApiClient.Setup(x => x.GetTrusteesFromSubmitted(applicationId)).ReturnsAsync(trusteesFromSubmitted);
+
+            var whosInControlFromSubmitted = new List<PersonInControl>
+            {
+                new PersonInControl
+                {
+                    Name = personInControlName+ whosInControlPostfix + sourceSubmitted,
+                    DateOfBirth = personInControlDob + whosInControlPostfix + sourceSubmitted
+                }
+            };
+            _organisationSummaryApiClient.Setup(x => x.GetWhosInControlFromSubmitted(applicationId)).ReturnsAsync(whosInControlFromSubmitted);
+
+
+            var request = new GetPeopleInControlRequest(applicationId, UserName);
+            var response = _orchestrator.GetPeopleInControlViewModel(request);
+
+            var viewModel = response.Result;
+
+            Assert.AreEqual(GatewayPageIds.PeopleInControl, viewModel.PageId);
```

Comment:
There's a huge amount of asserts here, is there a better way? Like could you write an equality function or something?

Reply:
Maybe. I might just break each area into it's own test, and head the whole setup to do the run once, if that makes sense.  I'll change it whatever tho